src/lib/zork/freedom-module.ts

```diff
@@ -41,6 +41,9 @@ var socksEndpoint :net.Endpoint = {
 // run on the same machine.
 var PORTS = [9000, 9010, 9020];
 
+// Number of active peerconnections.
```

Comment:
Just one thing: this reports the number of **active RtcToNet instances** - obviously that's what we want for benchmarking cloud servers but since we might some day need to report SockstoRtc too...how about renaming this command `getters`? ("the number of active getters connected to this server")